sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -124,6 +124,30 @@ task postCommitIT {
   }
 }
 
+task postCommitExamples {
+  dependsOn 'installGcpTest'
+  dependsOn ':sdks:python:sdist'
+  dependsOn ':runners:google-cloud-dataflow-java:worker:shadowJar'
+
+  def dataflowWorkerJar = project(":runners:google-cloud-dataflow-java:worker").shadowJar.archivePath
+
+  doLast {
+    def testOpts = basicPytestOpts + ["--numprocesses=8", "--dist=loadfile", "--collect-only"]
+    def argMap = [
+            "test_opts": testOpts,
+            "sdk_location": files(configurations.distTarBall.files).singleFile,
+            "worker_jar": dataflowWorkerJar,
```

Comment:
let's run Dataflow examples with runnerV2. worker_jar should not be necessary for that.

Reply:
Done!